### PR TITLE
-added simple ether address check (WIK)

### PR DESCRIPTION
**Pull request by Moejoe90**
Simple components or util's like this would you put them all in a general util file or specify them as components ?

### CODE REVIEW

**Comment by recmo** [comment]
This is not a React component but just a utility function. As such, it should start with a lowercase letter (only components and classes start with uppercase) and it should not have `propTypes` (which is also only for components.

I suggest we create a file `/src/ether-utils.js` similar to the `src/utils.js` that is already there.

File: src/components/EtherAddress.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import PropTypes from 'prop-types';
+import { isValidAddress } from 'ethereumjs-util';
+
+// Accepts checksummed addresses too
+const EtherAddress = address => isValidAddress(address);
```
Comment (recmo) [inline src/components/EtherAddress.js]: This does not return Html (but boolean) so it is not a React component but a regular function.

**Review by recmo — CHANGES_REQUESTED** [review]

**Comment by Moejoe90** [comment]
I fixed the comments waiting for confirmation

File: src/ether-utils.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { isValidAddress } from 'ethereumjs-util';
+
+// Accepts checksummed addresses too
+const EtherAddress = address => isValidAddress(address);
```
Comment (recmo) [inline src/ether-utils.js:4]: This is a regular function, not a component, so by convention it should start with a lowecase letter.

File: src/ether-utils.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+
+export default EtherAddress;
```
Comment (recmo) [inline src/ether-utils.js:6]: We want to add multiple functions to this file, so there is not a function that would obviously be default.

File: src/ether-utils.js (hunk 2 above)
Comment (recmo) [inline src/ether-utils.js:4]: The function name doesn't describe it's purpose. Something like `checkAddress`, `verifyAddress`, `validateAddress`, etc, would work.

**Review by recmo — APPROVED** [review]
Let's merge it as is and revisit later.

**Comment by Moejoe90** [comment]
I fixed these changes do we make a new pr for them, or just keep for now?

File: src/ether-utils.js (hunk 3 above)
Comment (Moejoe90) [reply src/ether-utils.js:6]: Es lint gives this error then 
4:1  error  Prefer default export  import/prefer-default-export
and the commit fails unless we put it in eslint ignore

File: src/ether-utils.js (hunk 2 above)
Comment (Moejoe90) [reply src/ether-utils.js:4]: I chose verifyAddress

**Pull request merged**